projects/fhi-angular-components/src/lib/fhi-popover-menu/fhi-popover-menu.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { RouterModule } from '@angular/router';
+
+import { Options } from '@popperjs/core';
+import { NgbPopoverModule } from '@ng-bootstrap/ng-bootstrap';
+
+import { FhiPopoverItem } from './fhi-popover-menu.model';
+
+@Component({
+  selector: 'fhi-popover-menu',
+  standalone: true,
+  imports: [CommonModule, NgbPopoverModule, RouterModule],
+  templateUrl: './fhi-popover-menu.component.html',
+})
+export class FhiPopoverMenuComponent {
+  @Input() items: Array<FhiPopoverItem>;
+  @Input() triggerIcon = 'three-dots-vertical';
+  @Output() actionEvent = new EventEmitter<string>();
+
+  popperOptions = (options: Partial<Options>) => {
```

Comment:
popoverOptions?

Reply:
Forget it, popperOptions is in use. I got confused since I thougt popper.js wasn't in use when we use Angular ;)